src/components/Review/ReviewSubmitButton.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react'
+import { Button, ModalProps } from 'semantic-ui-react'
+import { ModalWarning } from '..'
+import strings from '../../utils/constants'
+import { ReviewResponse, ReviewStatus } from '../../utils/generated/graphql'
+import { useRouter } from '../../utils/hooks/useRouter'
+import useSubmitReview from '../../utils/hooks/useSubmitReview'
+import messages from '../../utils/messages'
+import { AssignmentDetailsNEW, FullStructure } from '../../utils/types'
+
+const ReviewSubmitButton: React.FC<{
+  structure: FullStructure
+  reviewAssignment: AssignmentDetailsNEW
+  scrollTo: (code: string) => void
+}> = ({ scrollTo, structure, reviewAssignment }) => {
```

Comment:
Maybe better to define `interface` above to easy reading this
```suggestion
interface ReviewSubmitButtonProps {
  structure: FullStructure
  reviewAssignment: AssignmentDetailsNEW
  scrollTo: (code: string) => void
}
const ReviewSubmitButton: React.FC<ReviewSubmitButtonProps> = ({ scrollTo, structure, reviewAssignment }) => {
```

Reply:
agree, thanks for suggestion !